requirements.in

```diff
@@ -14,9 +14,10 @@ google-cloud-logging==2.7.0
 google-cloud-pubsub==2.18.3
 google-cloud-storage==1.43.0
 uvicorn==0.29.0
-fastapi[all]==0.110.2
+fastapi[all]==0.112.2
 strawberry-graphql[fastapi]==0.235.1
 databases[mysql]==0.9.0
+pymysql<=1.1.1
```

Comment:
Do we know why need to limit this one?

Reply:
So the official `databases` package has it pinned to `1.1.0` and they have an open PR to bump it to `1.1.1` but it has been open since May so who knows when that will be merged :(

Reply:
hm, I made a comment on this but it didn't come through, I think this should be `>=` right?

Reply:
I've just removed it for now, but I used `<=` just so we didn't have another instance of a core package getting a faulty update like `requests` did a while back